### PR TITLE
ci: py: flake8 fixes

### PR DESCRIPTION
**Pull request by blueyed**
Ref: https://github.com/neovim/neovim/pull/10631#issuecomment-515756591

### CODE REVIEW

**Comment by janlazo** [comment]
Did you run a fixer to enforce 4-space indent on all python files?

**Comment by blueyed** [comment]
I assume `autopep8` would have taken care of it - but it did not appear to be an issue.

File: scripts/check-includes.py [hunk 1]
```diff
@@ -9,58 +9,56 @@
...
-  argparser.add_argument('iwyu_args', nargs='*',
-                         help='IWYU arguments, must go after --.')
-  args = argparser.parse_args(argv)
+    argparser = ArgumentParser()
+    argparser.add_argument('--generated-includes-dir', action='append',
+                           help='Directory where generated includes are located.')
+    argparser.add_argument('--file', type=open, help='File to check.')
+    argparser.add_argument('iwyu_args', nargs='*',
+                           help='IWYU arguments, must go after --.')
+    args = argparser.parse_args(argv)
```
Comment (blueyed) [inline scripts/check-includes.py]: Oh.. here it fixed it to use 4 spaces indenting.

**Comment by justinmk** [comment]
If we're going to start using `autopep8` I would rather suggest [black](https://github.com/psf/black) (with single-quotes option; enforcing double-quotes is annoying).

**Comment by blueyed** [comment]
We're not using `autopep8` from now on - it was just a quick (and less intrusive) way to get it fixed.
I assume black would cause a larger diff.

File: scripts/gen_vimdoc.py [hunk 2]
```diff
@@ -287,12 +289,14 @@ def render_params(parent, width=62):
...
-                    indent=(' ' * len(name)))
+                                  indent=(' ' * len(name)))
 
         out += '{}{}\n'.format(name, desc)
     return out.rstrip()
 
 # Renders a node as Vim help text, recursively traversing all descendants.
+
+
 def render_node(n, text, prefix='', indent='', width=62):
```
Comment (justinmk) [inline scripts/gen_vimdoc.py]: the comment should go next to the function. I guess it should be a docstring.

File: scripts/gen_vimdoc.py [hunk 3]
```diff
@@ -317,7 +321,9 @@ def render_node(n, text, prefix='', indent='', width=62):
         text += ' [verbatim] {}'.format(get_text(n))
     elif n.nodeName == 'listitem':
         for c in n.childNodes:
-            text += indent + prefix + render_node(c, text, indent=indent+(' ' * len(prefix)), width=width)
+            text += indent + prefix + \
+                render_node(c, text, indent=indent +
+                            (' ' * len(prefix)), width=width)
```
Comment (justinmk) [inline scripts/gen_vimdoc.py]: autopep8 should put operators at start-of-line, not end.

File: src/clint.py [hunk 4]
```diff
@@ -2610,9 +2610,13 @@ def CheckBraces(filename, clean_lines, linenum, error):
...
             func_start_linenum = end_linenum + 1
-            while not clean_lines.lines[func_start_linenum] == '{':
-                attrline = Match(r'^((?!# *define).*?)(?:FUNC_ATTR|FUNC_API|REAL_FATTR)_\w+(?:\(\d+(, \d+)*\))?',
-                                 clean_lines.lines[func_start_linenum])
+            while not clean_lines.lines[func_start_linenum] == "{":
+                attrline = Match(
+                    r"^((?!# *define).*?)"
+                    r"(?:FUNC_ATTR|FUNC_API|REAL_FATTR)_\w+"
+                    r"(?:\(\d+(, \d+)*\))?",
+                    clean_lines.lines[func_start_linenum],
```
Comment (justinmk) [inline src/clint.py]: why double quotes?

**Review by justinmk — APPROVED** [review]

File: scripts/gen_vimdoc.py (hunk 3 above)
Comment (blueyed) [reply scripts/gen_vimdoc.py]: Used black (with single quotes) for that now.

File: src/clint.py (hunk 4 above)
Comment (blueyed) [reply src/clint.py]: Fixed.

**Pull request merged**